telemetry_core/src/main/java/com/newrelic/telemetry/transport/BatchDataSender.java

```diff
@@ -149,8 +149,9 @@ private Response sendPayload(byte[] payload)
       }
     } catch (IOException e) {
       logger.warn(
-          "IOException while trying to send data to New Relic. Batch retry recommended.", e);
-      throw new RetryWithBackoffException();
+          "IOException (message: {}) while trying to send data to New Relic. Batch retry recommended.",
+          e.getMessage());
+      throw new RetryWithBackoffException(e);
```

Comment:
Maybe nitpicky/subtle -- but the messaging inside the exception now suggests that the API recommended exponential backoff, which isn't really necessarily true in the generic IOException, right?

Reply:
Fair, although that hasn't actually changed in this case...it was doing it all along. :)

Reply:
Should the exception case have a different message, do you think?

Reply:
It probably does, something similar to what's being logged...just to avoid the incorrect suggestion about the API.

Reply:
ok, check it out now